python/dgllife/utils/complex_to_graph.py

```diff
@@ -140,27 +140,36 @@ def PN_graph_construction_and_featurization(ligand_mol,
         ligand_atom_indices_left = list(range(ligand_mol.GetNumAtoms()))
         protein_atom_indices_left = list(range(protein_mol.GetNumAtoms()))
 
-    # Construct bigraph for stage 1
-    node_featurizer = CanonicalAtomFeaturizer(atom_data_field='h')
-    edge_featurizer = CanonicalBondFeaturizer(bond_data_field='e')
+    # Node featurizer for stage 1
+    atoms = ['H','N','O','C','P','S','F','Br','Cl','I','Fe','Zn','Mg','Cs','Na','Mn','Ca','Co','Ni','Se','Cu','Cd','Hg','Sr','K']
+    atom_total_degrees = list(range(9))
+    atom_formal_charges = [-1, 0, 1]
+    atom_implicit_valence = list(range(5))
+    atom_explicit_valence = list(range(8))
+    atom_concat_featurizer = ConcatFeaturizer([partial(atom_type_one_hot, allowable_set=atoms), 
+                                               partial(atom_total_degree_one_hot, allowable_set=atom_total_degrees),
+                                               partial(atom_formal_charge_one_hot, allowable_set=atom_formal_charges),
```

Comment:
If we follow the order in `CanonicalAtomFeaturizer`, `atom_implicit_valence_one_hot` should be right after `atom_total_degree_one_hot`.